packages/server/src/__tests__/plugin/subscriptionCallback/index.test.ts

```diff
@@ -1635,6 +1840,7 @@ function mockRouterCheckResponse(opts?: {
     url?: string;
     id?: string;
     verifier?: string;
+    path?: string;
```

Comment:
Hmm in our examples it's not always using `/callback/:id` and I'm not really fan of processing to url and split the url to get the path because it introduces more logic in our tests and I prefer to keep very dumb tests. If you think it's better to write that logic let me know I'll do it

Reply:
Go ahead and keep as is, I misunderstood and made the assumption that the `/callback` path was not a configurable detail of the router.